stats/lib/routes.js

```diff
@@ -23,58 +23,64 @@ import {
+      reply.send(await fetchDailyDealStats(request.server.pg, request.filter))
     })
+    
+    
     app.get('/deals/summary', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchDealSummary(pgPools, request.filter))
+      reply.send(await fetchDealSummary(request.server.pg, request.filter))
     })
     app.get('/retrieval-success-rate', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchRetrievalSuccessRate(pgPools, request.filter))
+      reply.send(await fetchRetrievalSuccessRate(request.server.pg, request.filter))
     })
     app.get('/participants/daily', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchDailyParticipants(pgPools, request.filter))
+      reply.send(await fetchDailyParticipants(request.server.pg, request.filter))
     })
     app.get('/participants/monthly', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchMonthlyParticipants(pgPools, request.filter))
+      reply.send(await fetchMonthlyParticipants(request.server.pg, request.filter))
     })
     app.get('/participants/change-rates', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchParticipantChangeRates(pgPools, request.filter))
+      reply.send(await fetchParticipantChangeRates(request.server.pg, request.filter))
     })
     app.get('/participant/:address/scheduled-rewards', async (/** @type {RequestWithFilterAndAddress} */ request, reply) => {
-      reply.send(await fetchParticipantScheduledRewards(pgPools, request.filter, request.params.address))
+      reply.send(await fetchParticipantScheduledRewards(request.server.pg, request.filter, request.params.address))
     })
     app.get('/participant/:address/reward-transfers', async (/** @type {RequestWithFilterAndAddress} */ request, reply) => {
-      reply.send(await fetchParticipantRewardTransfers(pgPools, request.filter, request.params.address))
+      reply.send(await fetchParticipantRewardTransfers(request.server.pg, request.filter, request.params.address))
     })
     app.get('/miners/retrieval-success-rate/summary', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchMinersRSRSummary(pgPools, request.filter))
+      reply.send(await fetchMinersRSRSummary(request.server.pg, request.filter))
     })
     app.get('/miners/retrieval-timings/summary', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchMinersTimingsSummary(pgPools, request.filter))
+      reply.send(await fetchMinersTimingsSummary(request.server.pg, request.filter))
     })
     app.get('/retrieval-result-codes/daily', async (/** @type {RequestWithFilter} */ request, reply) => {
-      reply.send(await fetchDailyRetrievalResultCodes(pgPools, request.filter))
+      reply.send(await fetchDailyRetrievalResultCodes(request.server.pg, request.filter))
```

Comment:
```suggestion
      reply.send(await fetchDailyRetrievalResultCodes(request.server.pg.stats, request.filter))
```

We could also pass correct instance of the database instead of passing the pg object. In that case we don't need to change fetchers.

Reply:
ping @Goddhi

Reply:
ping @Goddhi